src/DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,12 @@ public function getConfigTreeBuilder()
                         ->scalarNode('server')->cannotBeEmpty()->end()
                         ->scalarNode('enabled')->defaultFalse()->end()
                         ->arrayNode('ip_white_list')
+                            ->prototype('scalar')->end()
+                            ->defaultValue(['127.0.0.1'])
```

Comment:
If there is a default value in both options, I guess you would always get an exception.

Reply:
If we provide a default value for one option, we will still have an exception when the other will be defined.

I'm not sure the exception is the right thing to do.
We could
- Remove the ipwhitelist default value
- If ipwhitelist is defined, use it. Else use trustediplist.

Reply:
Solved